libqtile/utils.py

```diff
@@ -30,6 +30,10 @@
 from random import randint
 from shutil import which
 
+from dbus_next import Message  # type: ignore
+from dbus_next.aio import MessageBus  # type: ignore
+from dbus_next.constants import BusType, MessageType  # type: ignore
+
 from libqtile.log_utils import logger
 
 _can_notify = False
```

Comment:
I've just noticed this section and the `send_notification` method further down. I assume we want to strip this out to use dbus_next again, rather than gi?

Reply:
My thinking here is just to have the `send_notification` make a dbus method call to `org.freedesktop.Notifications.Notify` as, that way, it should be able to work with any notification server that the user has running. Was hoping to sort tonight.

Reply:
Isn't that what it does already? (Assuming `_can_notify` is true). Given that we're making `dbus_next` a hard dependency, i agree it makes sense to strip out that `_can_notify` import test.

Reply:
Yes, I think it does but it does it via gi.repository.Notify and we're catching the ImportError. Given the dependency, I can remove the import test completely and rework the notification to use dbus_next instead.